### ISSUE TITLE
Tipping Banner should be more accessible

### ISSUE THREAD

**Issue by ryanml** [brave/brave-browser#6147]
<img width="749" alt="Screen Shot 2019-09-24 at 11 34 06 AM" src="https://user-images.githubusercontent.com/8732757/65539941-40df3300-debf-11e9-918a-a80c5b8d8eb8.png">

The following updates should be made:

1. Banner must receive focus upon opening.
2. ESC key must close banner.
3. Choice A button for BAT tip amount must be pre-selected upon banner open.
4. "SEND TIP" button changes shape as shown.
5. "SEND TIP" button is pre-focused so that RETURN key executes send action and closes banner window.
6. ALL 4 buttons present in this panel are set to receive focus for accessibility when tabbing through action elements.
7. Update default text to reflect copy shown in photo.

**Comment by NejcZdovc** [comment]
@davidtemkin I would go with mid option and not with lower option by default

**Comment by LaurenWags** [comment]
Questions have been added to the spec for @mandar-brave @alexwykoff to address in regards to item 3 and 7 from description.

**Comment by LaurenWags** [comment]
Verified passed with

Brave | 1.1.4 Chromium: 78.0.3904.97 (Official Build) beta (64-bit)
-- | --
Revision | 021b9028c246d820be17a10e5b393ee90f41375e-refs/branch-heads/3904@{#859}
OS | macOS Version 10.13.6 (Build 17G5019)

* Confirmed the banner receives focus upon opening.
* Confirmed ESC key closes the banner (when banner is in focus)
* Confirmed Choice B button for BAT tip amount is pre-selected upon banner open (per https://github.com/brave/brave-browser/issues/6147#issuecomment-534757269 )
* Confirmed `Send Tip` button has same shape as image in description.
* Confirmed when user has enough BAT to cover the tip preselected in Choice B, the `Send Tip` button is pre-focused and the return key executes send action and closes banner window. However, if you do not have enough BAT, there is no indication of what is in focus on the banner. Logged #6985 for this. Could be related to #6914 for monthly contribution banner.
* Confirmed all buttons present in this panel are set to receive focus for accessibility when tabbing through action elements. However, I did encounter and log #6986 for this. Once you tab away from `Send Tip` you are taken out of the banner and unable to return to the banner without clicking inside the banner with the mouse.
* Confirmed the default text was updated to reflect copy shown in image in description.
<img width="1239" alt="Screen Shot 2019-11-18 at 3 44 05 PM" src="https://user-images.githubusercontent.com/28145373/69092725-04d9d180-0a1b-11ea-8334-d3e93bf0e13d.png">
* Confirmed custom banners still display as expected. 

<img width="1248" alt="Screen Shot 2019-11-18 at 3 11 14 PM" src="https://user-images.githubusercontent.com/28145373/69092746-0dcaa300-0a1b-11ea-8f10-1c3cdb3f2d66.png">
